src/stream/mod.rs

```diff
@@ -29,6 +29,7 @@ pub use repeat_with::{repeat_with, RepeatWith};
 pub use stream::{
     Chain, Filter, Fuse, Inspect, Scan, Skip, SkipWhile, StepBy, Stream, Take, TakeWhile, Zip,
 };
+pub use successor::{successor, Successor};
```

Comment:
Let's rename "successor" to "successors" :)

```suggestion
pub use successors::{successors, Successors};
```